app/views/jobs/_output.html.erb

```diff
@@ -7,12 +7,11 @@
         <button class="btn btn-default" id="output-grow">Expand</button>
       </div>
 
-      <div class="pull-right only-finished">
-        <button class="btn btn-default" id="output-grow-toggle">Expand</button>
-        <%= link_to "Download log", project_job_path(@project, job, format: :text), class: "btn btn-primary" %>
-      </div>
+      <button class="btn btn-default only-finished" id="output-grow-toggle">Expand</button>
+      <%= link_to "Download log", project_job_path(@project, job, format: :text), class: "btn btn-primary only-finished" %>
+
       <% if job.can_be_stopped_by?(current_user) %>
-        <%= stop_button(class: "btn btn-danger pull-right only-active") %>
+        <%= stop_button(deploy: job, class: "btn btn-danger only-active") %>
```

Comment:
That's a little weird to pass in a job when it appears to be expecting an instance of `Deploy`, but I see what you're doing, and can't think of a better name for the parameter off the top of my head.